app/controllers/neighborhoods_controller.rb

```diff
@@ -9,12 +9,12 @@ def search
   end
 
   def show
```

Comment:
Hi, why did you comment out the show method?  It was probably unprofessional for me to push to github the p statements that I used to see params[:id] and id, my bad!  But was there something wrong with lines 16-17?  (Feel free to answer in person at hack night, by the way.)